src/main/java/io/jenkins/plugins/pipelinegraphview/multipipelinegraphview/MultiPipelineGraphViewAction.java

```diff
@@ -96,6 +81,6 @@ public String getUrlName() {
 
   @Override
   public String getIconClassName() {
-    return "icon-pipeline-graph";
+    return "symbol-options-outline plugin-ionicons-api";
```

Comment:
Stuggling to find connection between stages view and "options" icon as nothing gets configured there. Maybe "footsteps" or "layers" or "list" icon instead?

Reply:
Nice catch! I agree, the layers one fits better indeed.